manubot/process/metadata.py

```diff
@@ -157,3 +157,53 @@ def get_manuscript_urls(html_url: Optional[str] = None) -> dict:
             + "\n".join(x.url for x in response.history + [response])
         )
     return urls
+
+
+def get_software_versions():
+    from manubot import __version__ as manubot_version
+
+    return {
+        "manubot_version": manubot_version,
```

Comment:
This seems like a good starting point.  If you want to expand the options in the future, you could still keep `manubot_version` as is and add another key like `manubot_commit`.